.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/FineTuningTests.cs

```diff
@@ -213,19 +209,20 @@ public async Task CreateAndDeleteFineTuning()
             TrainingFile = uploadedFile.Id
         }.ToBinaryContent();
 
-        ClientResult result = await client.CreateJobAsync(requestContent);
-        FineTuningJob job = ValidateAndParse<FineTuningJob>(result);
+        CreateJobOperation operation = await client.CreateJobAsync(requestContent, false);
```

Comment:
nit for clarity:

```suggestion
        CreateJobOperation operation = await client.CreateJobAsync(requestContent, waitForCompletion: false);
```

makes it more readable